src/app/(main)/profile/page.tsx

```diff
@@ -73,36 +77,78 @@ export default function Page() {
                 />
               </div>
               <div>
-                <p className="text text-sm text-gray-500 mb-2">
+                <p className="text select-none text-sm text-gray-500 mb-2">
                   Change Password:
                 </p>
-                <Input
-                  type="password"
-                  placeholder="New Password"
-                  value={formData.password}
-                  onChange={(event) =>
-                    setFormData({
-                      ...formData,
-                      password: event.target.value,
-                    })
-                  }
-                />
+                <div className="relative z-0">
+                  <Input
```

Comment:
The `Input` component has a `right` property. Use that for the show password button.

Reply:
Had a hard time using relative and z-axis, looks like there exists a simpler way. Thanks.